### PR TITLE
Add main app

### PR DESCRIPTION
**Pull request by lsenta**

### CODE REVIEW

File: apps/main/main/src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component, HostListener } from '@angular/core';
+import { AuthQuery } from '@blockframes/auth';
+import { IconComponent } from '@blockframes/ui';
+
+@Component({
+  selector: 'delivery-root',
+  template: `
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/app.component.ts]: Format on one line.

File: apps/main/main/src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <router-outlet></router-outlet>`,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AppComponent {
+  constructor(private query: AuthQuery,
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/app.component.ts]: Format on one line.

File: apps/main/main/src/app/app.component.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+              private icons: IconComponent
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/app.component.ts]: Components shouldn't be injected. Only `Injectable` services should be.

File: apps/main/main/src/app/app.module.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  declarations: [AppComponent, LayoutComponent, HomeComponent],
+  imports: [
+    // Angular
+    BrowserModule,
+    BrowserAnimationsModule,
+    AppRoutingModule,
+    FlexLayoutModule,
+    HttpClientModule,
+
+    // Material
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/app.module.ts]: Do you need all thoses material stuff for 3 components (AppComponent, LayoutComponent, HomeComponent) ?

File: apps/main/main/src/app/home/home.component.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<section class="card-container">
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/home/home.component.html]: Use mat-card if this is supposed to looks like a card. Also use `fxLayout="row"` and remove all references to flex in the style.

File: apps/main/main/src/app/home/home.component.scss [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  box-sizing: border-box;
+  overflow: hidden;
+  width: auto;
+  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2);
+  padding: 30px;
+  text-align: center;
+  margin-top: 66px;
+}
+
+.card-container h2 {
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/home/home.component.scss]: You shouldn't style the font. Use material typography instead: https://material.angular.io/guide/typography

File: apps/main/main/src/app/home/home.component.ts [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+export class HomeComponent implements OnInit {
+
+  constructor() {}
+
+  ngOnInit() {
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/home/home.component.ts]: remove ngOnInit and constructor if you don't use them.

File: apps/main/main/src/app/app.component.ts (hunk 1 above)
Comment (lsenta) [reply apps/main/main/src/app/app.component.ts]: delivery code, to be removed with @ioaNikas next patch

File: apps/delivery/delivery/src/app/delivery.module.ts [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule,
+    RouterModule.forChild(routes)
+  ],
+  providers: [],
+})
+export class DeliveryAppModule {}
```
Comment (GrandSchtroumpf) [inline apps/delivery/delivery/src/app/delivery.module.ts]: Maybe `DeliverySubAppModule` ?

File: apps/delivery/delivery/src/main.ts [hunk 9]
```diff
@@ -28,5 +28,5 @@ if (environment.production) {
 }
 
 platformBrowserDynamic()
-  .bootstrapModule(AppModule)
+  .bootstrapModule(DeliveryAppModule)
```
Comment (GrandSchtroumpf) [inline apps/delivery/delivery/src/main.ts]: It won't work. You need to create another Module that bootstrap the app.

File: apps/ip-upload/ip-upload/src/app/app.module.ts [hunk 10]
```diff
@@ -31,6 +33,7 @@ import { AccountModule, ProfileModule } from '@blockframes/account';
   declarations: [AppComponent, LayoutComponent],
   imports: [
     BrowserModule,
+    HttpClientModule,
```
Comment (GrandSchtroumpf) [inline apps/ip-upload/ip-upload/src/app/app.module.ts]: Why ?

File: apps/main/main/src/app/home/home.component.html [hunk 11]
```diff
@@ -0,0 +1,22 @@
+<section>
```
Comment (GrandSchtroumpf) [inline apps/main/main/src/app/home/home.component.html]: section is useless here. Remove it.

**Comment by Jackseed** [comment]
Ref #1244

**Comment by Jackseed** [comment]
Upgrades to come but close this one for now

**Pull request closed without merging**